src/PrestaShopBundle/Resources/views/Admin/Improve/Shipping/Carriers/form.html.twig

```diff
@@ -35,5 +35,11 @@
 
 {% block javascripts %}
   {{ parent() }}
-  <script src="{{ asset('themes/new-theme/public/carrier_form.bundle.js') }}"></script>
+  <script src="{{ asset('themes/new-theme/public/carrier_form.bundle.js') }}">
+  </script>
+  <script>
+    $(document).ready(function() {
+      $('.select2').select2();
```

Comment:
Shouldn't this be in the bundle?

Reply:
I'm getting some help from @mattgoud on the js and css parts. Could you tell me where to put this so I is in the right place ? Thx

Reply:
@tleon I think that this should be the right file - https://github.com/PrestaShop/PrestaShop/blob/990e1eb237f10e79dcae1f5d3104ad62cb6e694c/admin-dev/themes/new-theme/js/pages/carrier/form/index.ts#L31 👍 

If you put `$('.select2').select2();` into `$(() => {` after the init, it should work